src/psyclone/dynamo0p3.py

```diff
@@ -3437,14 +3462,7 @@ def _compute_halo_depth(self):
         # if there is only one entry in the list we can just return
         # the depth
         if len(depth_info_list) == 1:
-            depth_info = depth_info_list[0]
-            if depth_info.max_depth:
-                # return the maximum halo depth (which is returned by
-                # calling get_halo_depth with no depth argument)
-                # TODO fix hard-wired mesh name here
-                return "mesh%get_halo_depth()"
-            else:  # return the variable and/or literal depth expression
-                return str(depth_info)
+            return str(depth_info_list[0])
```

Comment:
pylint points out the the following `else:` is now redundant because of the `return`

Reply:
Changed to make pylint happy.